README.md

```diff
@@ -37,39 +35,101 @@ group :development, :test do
 end
 ```
 
-Generators for factories will automatically substitute fixture (and maybe any other
-`fixture_replacement` you set). If you want to disable this feature, add the
-following to your application.rb file:
+You may want to configure your test suite to include factory\_bot methods; see
+[configuration](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#configure-your-test-suite).
+
+### Automatic Factory Definition Loading
+
+By default, factory\_bot\_rails will automatically load factories
+defined in the following locations,
+relative to the root of the Rails project:
+
+```
+factories.rb
+test/factories.rb
+spec/factories.rb
+factories/*.rb
+test/factories/*.rb
+spec/factories/*.rb
+```
+
+You can configure this with:
+
+```ruby
+config.factory_bot.definition_file_paths = ["custom/factories"]
```

Comment:
Does it make sense to indicate what file this should be placed in, or is it clear/mentioned elsewhere?